tests/nnstreamer_if/unittest_if.cc

```diff
@@ -648,6 +648,98 @@ TEST (tensor_if_appsrc, action_0)
   g_free (str_pipeline);
 }
 
+/**
+ * @brief custom callback function
+ */
+gboolean tensor_if_custom_cb (const GstTensorIf * tensor_if,
+    const GstTensorMemory * input, gboolean * result) {
+  gint *output, i;
+
+  output = (gint *) input->data;
+
+  for (i = 0; i < 48; i++) {
+    if (test_frames[0][i] != output[i]) {
+      *result = FALSE;
+    }
+  }
+  *result = TRUE;
```

Comment:
It seems that `result` always goes to `TRUE`.

Reply:
It's intended.